etc/kayobe/ansible/monasca_ceph_monitoring.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Deploy Monasca-rsyslog on Ceph nodes
+  hosts: ceph
+  become: true
+  roles:
+  - role: stackhpc.monasca-rsyslog
+    monasca_rsyslog_api_auth:
+      auth_url: "http://{{ ilab_vip_address }}:5000"
+      project: "monasca"
+      username: "monasca-agent"
+      password: "{{ secrets_monasca_agent_password }}"
+      region_name: "RegionOne"
+      endpoint_type: "public"
+    monasca_rsyslog_venv: "/usr/libexec/monasca-rsyslog"
+
+- name: Deploy Monasca Agent
+  hosts: ceph
+  become: true
+  roles:
+  - role: stackhpc.monasca-agent
+    keystone_url: "http://{{ ilab_vip_address }}:5000/v3"
+    monasca_user: root
+    monasca_agent_user: "monasca-agent"
+    monasca_agent_password: "{{ secrets_monasca_agent_password }}"
+    monasca_agent_project: "monasca"
+    monasca_agent_git_repo: https://github.com/stackhpc/monasca-agent
+    monasca_agent_branch: story-2005032
```

Comment:
We shouldn't merge it like this - once that change is tested we can create a PR and merge it in our fork. We should also push the change upstream once we know it works.